db/migrate/20150207203328_create_blacklisted_passwords.rb

```diff
@@ -0,0 +1,18 @@
+class CreateBlacklistedPasswords < ActiveRecord::Migration
+  def up
+    create_table :blacklisted_passwords do |t|
+      t.string :string
+
+      t.timestamps
+    end
+    add_index :blacklisted_passwords, :string, using: :hash
+
+    passwords = File.readlines(File.join(__dir__, '../password_blacklist.txt'))
+      .map(&:strip)
```

Comment:
We can and I did!

It's certainly a matter of taste but I myself prefer keeping distinct transformations separate as it makes it easier to see what it is that each of them does individually rather than having them all squeezed together. But that's me! :)